src/extension/tests/helpers/test.settings

```diff
@@ -0,0 +1,20 @@
+{
+    "runtimeSettings": [{
+        "handlerSettings": {
+            "protectedSettingsCertThumbprint": "",
+            "protectedSettings": "",
+            "publicSettings": {
+                "operation": "Installation",
+                "activityId": "12345-2312-1234-23245-32112",
+                "startTime": "2019-07-20T12:12:14Z",
+                "maximumDuration": "PT2H",
+                "rebootSetting": "IfRequired",
+                "classificationsToInclude":["Critical","Security"],
+                "patchesToInclude": ["*ern*=1.2*", "kern*=1.23.45"],
+                "patchesToExclude": ["test", "*test"],
+                "internalSettings": "test",
+                "patchRolloutId": "2019-07-22T12:12:14Z"
```

Comment:
maintenanceRunId

Reply:
This appears to be deliberate -- if so name files with more specificity as to their reason for existence and intended use. back_compat_test.settings?